src/irc/core/irc-servers.c

```diff
@@ -409,7 +423,10 @@ static void sig_disconnected(IRC_SERVER_REC *server)
                         server_redirect_destroy(tmp->next->data);
 	}
 	g_slist_free(server->cmdqueue);
-        server->cmdqueue = NULL;
+	server->cmdqueue = NULL;
+
+	g_slist_free_full(server->cap, g_free);
```

Comment:
>that's obviously not true

I take that back. I can't find any symbols from newer glib releases. This also means that using `g_base64_encode()` below would bump it up to 2.12 (released 2006, included in centos 5, debian lenny 5.0, ubuntu 8.04, so i'm pretty sure nobody is going to complain about that one)

`g_slist_free_full()` is definitely too recent though, and it's not too hard to replace it with a loop.